en/getting_started/flight_modes.md

```diff
@@ -2,7 +2,7 @@
 
 Flight modes define how the autopilot responds to remote control input, and how it manages vehicle movement during fully autonomous flight.
 
-The modes provide different types/levels of autopilot assistance to the user (pilot), ranging from automation of common tasks like takeoff and landing, through to mechanisms that make it easier to regain level flight, hold the vehicle to a fixed path or position, etc.
+The modes provide different types/levels of autopilot assistance to the user (pilot). This assistance ranges from automation of common tasks like takeoff and landing, mechanisms that make it easier to regain level flight, hold the vehicle to a fixed path or position, etc.
```

Comment:
FYI This doc all still to be done. MOst of this material moving into the flight-specific versions so this will have to become a linking doc with only high level summary.